### PR TITLE
3 feedback for v013

### PR DESCRIPTION
**Pull request by bcstrongx**
address issue #3

### CODE REVIEW

File: body.adoc [hunk 1]
```diff
@@ -252,7 +252,26 @@ _Exposing the WRPTR may also allow support for Linux perf's https://lwn.net/Arti
...
 === Supervisor Control Transfer Record Status (sctrstatus)
 
-The sctrstatus register is an S-mode and VS-mode (writable) alias to mctrstatus.  The CSR number is 0x183.
+The sctrstatus register is an S-mode and VS-mode (writable) alias to mctrstatus.
+
+=== CSR Listing
+
+[width="100%",cols="^12%,18%,70%",options="header",]
+|===
+| CSR Number | Name | Description
```
Comment (snehasish) [inline body.adoc:261]: nit: Sort these by CSR Number?

File: body.adoc [hunk 2]
```diff
@@ -520,20 +525,19 @@ while executing in an enabled privilege mode, and when FROZEN=0.
...
-(2^12^ + CCM) << CCE-1
+CCE==0? CCM : ((2^12^ + CCM) << CCE-1)
 ----
 [WARNING]
 ====
 _The 2^12^ is just a way to represent the leading 1.  Let me know if there's a better way to describe this._
 ====
 
-When CCE>1, the cycle count granularity is reduced from 1 to 2^CCE-1^.
+When CCE<=1, including cases where 13 or fewer CC bits are implemented, the CC field provides a simple binary count of elapsed cycles.  When CCE>1, the cycle count granularity is reduced from 1 to 2^CCE-1^.
```
Comment (snehasish) [inline body.adoc:559]: Should this be "the CCM field"?

**Review by snehasish — APPROVED** [review]
Looks good, only a couple of minor nits.

File: body.adoc (hunk 2 above)
Comment (bcstrongx) [reply body.adoc:559]: Oops, meant to remove that text, that was an earlier attempts at clarifying the behavior.  I think the equation now covers everything, so will remove it.

File: body.adoc (hunk 1 above)
Comment (bcstrongx) [reply body.adoc:261]: sure

**Pull request merged**